app/domain/domain.scala

```diff
@@ -26,7 +26,7 @@ object Platform {
 
   private val WindowsMinGWPattern = "(mingw|msys).*".r
 
-  private val LinuxPattern = "(linux|linux32|linux64)".r
+  private val LinuxPattern = "(linux|linux32|linux64|linuxarm32|linuxarm64)".r
```

Comment:
This regex will be working on the `platformId` that is received as request param from `sdkman-install.sh`. This will be a lowercase version of `SDKMAN_PLATFORM`.

Reply:
Never mind, I just checked and `linuxarm64` is correct :-)